python/cudf_polars/cudf_polars/testing/asserts.py

```diff
@@ -19,10 +19,13 @@
 
 __all__: list[str] = ["assert_gpu_result_equal", "assert_ir_translation_raises"]
 
+DEFAULT_GPU_ENGINE: GPUEngine = GPUEngine(raise_on_fail=True)
+
 
 def assert_gpu_result_equal(
     lazydf: pl.LazyFrame,
     *,
+    engine: GPUEngine = DEFAULT_GPU_ENGINE,
```

Comment:
Can we shovel this in to the function? That is, `engine: GPUEngine | None = None` and `engine = GPUEngine(raise_on_fail=True) if engine is None else engine`
?